chapter_convolutional-modern/vgg_vn.md

```diff
@@ -113,7 +119,7 @@ conv_arch = ((1, 64), (1, 128), (2, 256), (2, 512), (2, 512))
 The following code implements VGG-11. This is a simple matter of executing a for loop over `conv_arch`.
 -->
 
-*dịch đoạn phía trên*
+Đoạn mã nguồn sau đây sử dụng VGG 11. Việc này khá đơn giản khi thực hiện với vòng lặp for trên `conv_arch`.
```

Comment:
Đoạn mã nguồn sau đây `cài đặt mạng` VGG 11. Việc này khá đơn giản khi thực hiện với vòng lặp for trên `conv_arch`.

Reply:
```suggestion
Đoạn mã nguồn sau đây xây dựng mạng VGG 11. Việc này chỉ đơn giản là thực hiện vòng lặp `for` trên  biến `conv_arch`.
```
@davidnvq từ implement nhóm thống nhất chỉ dịch theo kiểu `lập trình` , `xây dựng` thôi nhé. `Cài đặt` sẽ cho `setup, install`.